ground/gcs/src/plugins/scope/scopegadgetoptionspage.cpp

```diff
@@ -91,20 +91,36 @@ QWidget* ScopeGadgetOptionsPage::createPage(QWidget *parent)
     ExtensionSystem::PluginManager *pm = ExtensionSystem::PluginManager::instance();
     UAVObjectManager *objManager = pm->getObject<UAVObjectManager>();
     QVector< QVector<UAVDataObject*> > objList = objManager->getDataObjectsVector();
+    QStringList toAdd;
+
+    QString previous = NULL;
+
     foreach (QVector<UAVDataObject*> list, objList) {
         foreach (UAVDataObject* obj, list) {
+	    QString name = obj->getName();
+
             if (obj->isSingleInstance())
             {
-                options_page->cmbUAVObjects->addItem(obj->getName());
+		toAdd.append(name);
             }
-            else if(obj->getName() != options_page->cmbUAVObjects->itemText(options_page->cmbUAVObjects->count()-1))
+            else if(name != previous)
```

Comment:
Not part of this PR I know but why are multi instance object limited to a single instance here?